relay-event-normalization/src/lib.rs

```diff
@@ -24,13 +24,12 @@ mod legacy;
 mod normalize;
 mod regexes;
 mod remove_other;
-pub mod replay;
-mod schema;
 mod statsd;
 mod timestamp;
 mod transactions;
 mod trimming;
 
+pub mod replay;
```

Comment:
Why has this been moved to imports instead of, as above, modules definitions?  I think this should stay where it was before.

Reply:
In the codebase we separate `mod`s and `pub mod`s. Fwiw we also separate `pub mod`s and `pub use`s, which I haven't done in this PR, so happy to create a new section. What do you think?